### PR TITLE
Update characteristics page

### PR DESCRIPTION
**Pull request by mikearildbrown**
Includes a round of edits from myself, @sion-docs and a third-party proofreading service.

### CODE REVIEW

**Comment by james-nash** [comment]
FYI, just pushed a fix for the failing lint test.

File: site/docs/foundations/characteristics/index.mdx [hunk 1]
```diff
@@ -6,42 +6,46 @@ sidebar:
...
 
-Characteristics are semantic groupings of visual attributes that are applied across relevant components. They drive consistency and support theming across complex libraries. They help to define the rules that style each component and aid the cohesiveness of the UI.
+## Benefits
 
-### Why characteristics?
+For designers, characteristics enable visual changes without impacting the efficiency or effectiveness of components.
 
-From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without the overhead of having to remember details such as hex values. Any time you want to tweak the value of a characteristic, you only have to do it in one place for it to reflect everywhere.
+For developers, they offer an easy way to reuse the same values. It creates a single point of reference for commonly used values, which means you only have to update in one place for it to reflect everywhere.
```
Comment (james-nash) [inline site/docs/foundations/characteristics/index.mdx:23]: Hmm... the first line about designers feels a bit woolly. What's it actually saying? It's not clear how visual changes without tokens impact the "efficiency or effectiveness of components".

As for the second point - that benefit also applies to designers.

The following bullet list summarises the benefits quite well IMHO, so perhaps these 2 paragraphs aren't needed at all?

That being said, one additional benefit I'd call out (perhaps as an additional bullet below?) is that (characteristic) design tokens create a shared language between designers, devs and other stakeholders - they all use the same names to refer to the same aspects/concepts in Salt-based UIs. That helps avoid miscommunications and streamlines design-to-dev handovers too.

**Review by james-nash — CHANGES_REQUESTED** [review]
Mostly looks good to me, but I think the "Benefits" section might need some tweaks.

File: site/docs/foundations/characteristics/index.mdx [hunk 2]
```diff
@@ -6,42 +6,46 @@ sidebar:
...
 
 Characteristics offer the following benefits:
```
Comment (libertymayc) [inline site/docs/foundations/characteristics/index.mdx:25]: we got rid of bullets for below points, but added for the bullets below? @mikearildbrown

File: site/docs/foundations/characteristics/index.mdx [hunk 3]
```diff
@@ -6,42 +6,46 @@ sidebar:
...
 
-- Ability to theme the Salt inline with third-parties or CIB sub brand guidelines.
-- Increase consistency across all components, with the knock-on effect of improving the overall user experience e.g. reduced cognitive load.
-- Simplified design of new components and patterns e.g. drag-and-drop behavior.
-- Streamlined evolution – a single change to a characteristic will automatically cascade down through all related components within the library.
+- Ability to theme Salt in line with third parties or sub-brand guidelines
+- Increased consistency across all components, improving the overall user experience, e.g., reduced cognitive load
+- Simplified design of new components and patterns e.g., drag-and-drop behavior
+- Streamlined design changes, e.g., a characteristic tweak automatically applying to all related components
```
Comment (libertymayc) [inline site/docs/foundations/characteristics/index.mdx:30]:
```suggestion
- Single reference points for commonly used values
- Streamlined design changes, e.g., a characteristic tweak automatically applying to all related components
- Shared language enabling smooth handovers from design to development
```

File: site/docs/foundations/characteristics/index.mdx (hunk 2 above)
Comment (libertymayc) [reply site/docs/foundations/characteristics/index.mdx:25]: sorry, i meant we got rid of the full stops on these bullet points, but added them to the below ones

Comment (libertymayc) [reply site/docs/foundations/characteristics/index.mdx:25]: below, bullet point full stops removed end of each sentence

- Ability to theme the Salt inline with third-parties or CIB sub brand guidelines.	- Ability to theme Salt in line with third parties or sub-brand guidelines
- Increase consistency across all components, with the knock-on effect of improving the overall user experience e.g. reduced cognitive load.	- Increased consistency across all components, improving the overall user experience, e.g., reduced cognitive load
- Simplified design of new components and patterns e.g. drag-and-drop behavior.	- Simplified design of new components and patterns e.g., drag-and-drop behavior
- Streamlined evolution – a single change to a characteristic will automatically cascade down through all related components within the library.	- Streamlined design changes, e.g., a characteristic tweak automatically applying to all related components

line ~36, full stops added
1. Always begin with `--salt-`.
2. Follow with the foundation name, e.g., `--salt-actionable-`, `--salt-text-`, `--salt-focused-`.
3. Append the variant if applicable, e.g., `--salt-actionable-cta-`, `--salt-text-h2-`.
4. Append the attribute, e.g., `--salt-actionable-cta-background`, `--salt-text-h2-fontSize`, `--salt-focused-outlineColor`.
5. Append the state if applicable, e.g., `--salt-actionable-cta-background-hover`.

**Comment by libertymayc** [comment]
what's happening with this PR @mikearildbrown

**Review by james-nash — APPROVED** [review]
👍

**Comment by pseys** [comment]
I'm taking a look at this now, but with interruptions its taking me time to get through it. I'll update with my feedback as soon as possible.

**Pull request merged**